apps/expo-go/src/navigation/Navigation.tsx

```diff
@@ -235,17 +236,18 @@ export default (props: { theme: ColorTheme }) => {
               <RootStack.Screen
                 name="Account"
                 component={AccountModal}
-                options={({ route, navigation }) => ({
+                options={({ route: _route, navigation: _navigation }) => ({
                   headerShown: false,
                   ...(Platform.OS === 'ios' && {
                     gestureEnabled: true,
                     cardOverlayEnabled: true,
-                    headerStatusBarHeight:
-                      navigation
-                        .getState()
-                        .routes.findIndex((r: RouteProp<any, any>) => r.key === route.key) > 0
-                        ? 0
-                        : undefined,
+                    // NOTE(brentvatne): it is unclear what this was intended for, it doesn't appear to be needed?
+                    // headerStatusBarHeight:
+                    //   navigation
+                    //     .getState()
+                    //     .routes.findIndex((r: RouteProp<any, any>) => r.key === route.key) > 0
+                    //     ? 0
+                    //     : undefined,
```

Comment:
kind of unrelated to this PR but i couldn't figure out what this was for and removing it appeared to (perhaps by chance!) fix some visual glitches in the header